sim/model_export.py

```diff
@@ -125,23 +125,19 @@ def get_init_buffer(self) -> Tensor:
 
     def forward(
         self,
-        x_vel: Tensor,  # x-coordinate of the target velocity
-        y_vel: Tensor,  # y-coordinate of the target velocity
-        rot: Tensor,  # target angular velocity
+        vector_command: Tensor,  # (x_vel, y_vel, rot)
```

Comment:
change the names to:
```suggestion
    JOINT_POSITIONS = "joint_positions"
    JOINT_VELOCITIES = "joint_velocities"
    VECTOR_COMMAND = "vector_command"
    IMU = "imu"
    TIMESTAMP = "timestamp"
    JOINT_TORQUES = "joint_torques"
```

Reply:
Do you want to change the names and order as well?